packages/components/src/EmptyState/EmptyState.module.css

```diff
@@ -0,0 +1,114 @@
+.container {
+  --empty-state-border-width: var(--border-width-1);
+  --empty-state-illustration-max-height: 366px;
+  --empty-state-illustration-min-width: 224px;
+  --empty-state-text-container-max-width: 426px;
+
+  container-type: inline-size;
+  width: 100%;
+  border: var(--empty-state-border-width) solid var(--empty-state-border-color);
+  border-radius: var(--border-solid-border-radius);
+  background-color: var(--empty-state-background-color);
+  color: var(--color-purple-800);
+}
+
+.straightCorners {
+  border-radius: 0;
+}
+
+/** @deprecated */
+.positive {
+  --empty-state-border-color: var(--color-green-500);
+  --empty-state-background-color: var(--color-green-100);
+}
+
+.negative,
+.action {
+  --empty-state-border-color: var(--color-red-500);
+  --empty-state-background-color: var(--color-red-100);
+}
+
+.neutral {
+  --empty-state-border-color: var(--color-purple-400);
+  --empty-state-background-color: var(--color-purple-100);
+}
+
+/** end @deprecated */
+.success {
+  --empty-state-border-color: var(--color-green-500);
+  --empty-state-background-color: var(--color-green-100);
+}
+
+.warning {
+  --empty-state-border-color: var(--color-red-500);
+  --empty-state-background-color: var(--color-red-100);
+}
+
+.informative {
+  --empty-state-border-color: var(--color-blue-400);
+  --empty-state-background-color: var(--color-blue-100);
+}
+
+.expert-advice {
+  --empty-state-border-color: var(--color-purple-400);
+  --empty-state-background-color: var(--color-purple-100);
+}
+
+.content {
+  justify-content: center;
+  display: grid;
+  grid-template-columns: minmax(var(--empty-state-illustration-min-width), auto) minmax(
+      auto,
+      var(--empty-state-text-container-max-width)
+    );
+  grid-template-rows: minmax(auto, var(--empty-state-illustration-max-height)) auto;
+  padding: var(--spacing-24);
+  grid-column-gap: var(--spacing-32);
```

Comment:
Can't screenshot this cause figma is dumb but I think the original spacing was 36px gap between the image in the designs - small nit but just want to confirm the update was intended / hunky dory

Reply:
(noting that this makes sense to me since it aligns to our new spacing token values)

Reply:
Haha you got me! Totally forgot about this.
I changed it because we don't actually have a `spacing-36` 👀 There's only 32 and 40. Let me check with Jess.

Reply:
Confirmed with Jess that 32 is the reasonable one to use 😄